### PR TITLE
Revisione schermata editing fix vari

### PR DESCRIPTION
**Pull request by taba90**
Provides several fixes to rndt metadata editor.
See issues:

- https://github.com/geosolutions-it/iso19139.rndt/issues/19

- https://github.com/geosolutions-it/iso19139.rndt/issues/20

- https://github.com/geosolutions-it/iso19139.rndt/issues/21

- https://github.com/geosolutions-it/iso19139.rndt/issues/22

- https://github.com/geosolutions-it/iso19139.rndt/issues/23

- https://github.com/geosolutions-it/iso19139.rndt/issues/24

- https://github.com/geosolutions-it/iso19139.rndt/issues/25

- https://github.com/geosolutions-it/iso19139.rndt/issues/27

- https://github.com/geosolutions-it/iso19139.rndt/issues/30

### CODE REVIEW

File: src/main/plugin/iso19139.rndt/loc/ita/labels.xml [hunk 1]
```diff
@@ -28,49 +28,48 @@
...
-		<description>Valore dalla lista suggerito dal DM, o numero intero indicante un CRS con codifica EPSG</description>
+    <description>Elenco dei codici per le 24 lingue ufficiali della UE</description>
+    <helper>
+      <option value="bul" >Bulgaro</option>
+      <option value="cze" >Ceco</option>
+      <option value="hrv" >Croato</option>
+      <option value="dan" >Danese</option>
+      <option value="est" >Estone</option>
+      <option value="fin" >Finlandese</option>
+      <option value="Francese" >fre</option>
```
Comment (etj) [inline src/main/plugin/iso19139.rndt/loc/ita/labels.xml]: value e label sembrano invertiti

File: src/main/plugin/iso19139.rndt/present/csw/csw-full.xsl [hunk 2]
```diff
@@ -173,12 +173,12 @@
         <xsl:choose>
             <xsl:when test="$isTile">
```
Comment (etj) [inline src/main/plugin/iso19139.rndt/present/csw/csw-full.xsl]: Rimuovere tutta la gestione dei Tile

File: src/main/plugin/iso19139.rndt/loc/ita/codelists.xml [hunk 3]
```diff
@@ -29,6 +29,34 @@
...
+        riesaminata e migliorata o emendata
+      </description>
+    </entry>
+    <!-- - - - - - - - - - - - - - - - - - - - - - - - - -->
+    <entry>
+      <code>rilievo</code>
+      <label>Rilievo</label>
+      <description>Data che identifica l’osservazione del fenomeno (es. volo)</description>
+    </entry>
+  </codelist>
```
Comment (etj) [inline src/main/plugin/iso19139.rndt/loc/ita/codelists.xml]: Non mi sembra corrispondano :)

https://ies-svn.jrc.ec.europa.eu/attachments/download/2469/eReporting_PriorityDataList_V2.0.pdf

**Review by etj — CHANGES_REQUESTED** [review]
Per la #24 mancano le modifiche a tutti i template:

https://github.com/geosolutions-it/iso19139.rndt/blob/rndt2/src/main/plugin/iso19139.rndt/templates/iso-19139.rndt-template-vector.xml#L68-L73

File: src/main/plugin/iso19139.rndt/present/csw/csw-full.xsl (hunk 2 above)
Comment (taba90) [reply src/main/plugin/iso19139.rndt/present/csw/csw-full.xsl]: fixed

File: src/main/plugin/iso19139.rndt/loc/ita/codelists.xml (hunk 3 above)
Comment (taba90) [reply src/main/plugin/iso19139.rndt/loc/ita/codelists.xml]: fixed

File: src/main/plugin/iso19139.rndt/loc/ita/labels.xml (hunk 1 above)
Comment (taba90) [reply src/main/plugin/iso19139.rndt/loc/ita/labels.xml]: fixed

**Pull request merged**